app/javascript/styles/tables.scss

```diff
@@ -9,13 +9,13 @@ table.border-side {
     text-transform: uppercase;
     letter-spacing: 1.5px;
     background: $light-gray;
-    padding-left: 0px !important;
+    padding: 0.8rem 0.5rem !important;
```

Comment:
The styling changes here affect a few other tables, including the call list dashboard, so I think we'll need to scope this a smidge tighter. (Def not your fault or anything; our CSS is the toughest part of our codebase to work in.) I would suggest slapping a new class on the accountants table, and adding a new CSS rule that applies these padding rules to it.

Reply:
All your `table.border-side th` are belong to us